29.10.22/js/average.js

```diff
@@ -0,0 +1,11 @@
+const userNumber1 = prompt('First number')
+const userNumber2 = prompt('Next number')
+const userNumber3 = prompt('Last number')
+
+const num1 = Number(userNumber1)
```

Comment:
You can combine methods to avoid redundant variables. This is bad for performance(